AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -466,6 +465,12 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)recursivelyFetchData;
 
 /**
+ * @abstract Triggers a call to fetchData for nodes in the preload range
```

Comment:
Let's be more specific here, and say that is a node has an interface state that specifies it is in the fetch data state. Then sometime after this call, the fetch data method will be called.

Note that the terminology has changed a bit, And I've dropped a turn preload in favor of fetch data. This is because preload is too generic of the term and can be misunderstood as preparing display as well.